client/pom.xml

```diff
@@ -27,10 +27,6 @@
     <artifactId>rocketmq-client</artifactId>
     <name>rocketmq-client ${project.version}</name>
 
-    <properties>
```

Comment:
Do not modify compiler level. we need to keep to be compatible with JDK 1.6 in SDK